dkg/sync/client.go

```diff
@@ -142,7 +142,7 @@ func (c *Client) isConnected() bool {
 
 // sendMsgs sends period sync protocol messages on the stream until error or shutdown.
 func (c *Client) sendMsgs(ctx context.Context, stream network.Stream) (bool, error) {
-	timer := time.NewTicker(time.Second)
+	timer := time.NewTicker(time.Second * 10)
```

Comment:
had to increase period messages to more than 1s, otherwise bad temp addresses are never removed from the peer store.